configs/nextflow.config

```diff
@@ -262,6 +279,22 @@ process {
     withLabel:time_queue_from_week {
         time = { check_max( escalate_queue_time( 'week', task ), 'time' ) }
     }
+
+    withLabel:time_queue_from_small2 {
```

Comment:
```suggestion
    withLabel:slow_time_queue_from_small {
```

Reply:
Format like that maybe?

Reply:
I thought I'd tag 2 in the label as in principle we could make it increment even more slowly like every 3 retries...
but I agree it's not pretty.
what about `time_queue_from_small_by2` ? or `time_queue_from_small_slow2` ?

Reply:
I'd have to transpose whatever label name choice into RVI pipeline

Reply:
My preference is `time_queue_from_small_by2`

Reply:
and it should be `escalate_queue_time_by2` too :)